src/index.html

```diff
@@ -1,27 +1,356 @@
 <!doctype html>
-<html lang="en">
+<html lang="pt-BR">
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Catalog</title>
     <link
-      rel="preconnect"
-      href="https://fonts.gstatic.com"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
     />
     <link
-      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
       rel="stylesheet"
+      href="index.scss"
```

Comment:
The stylesheet link is pointing to `index.scss`, which is incorrect. It should point to `index.css` to ensure the styles are applied correctly.